src/components/Molecules/CardItem/CardItem.css

```diff
@@ -117,7 +108,71 @@ div.blurb {
 }
 
 .icon {
-  left: 3px;
-  position: absolute;
-  top: 1px;
+  width: 1em;
+  height: 1em;
+  vertical-align: top;
+  margin-right: 0.25em;
+}
+
+@media bp-phone {
+  .cardItem.normal {
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: 40% auto;
+    grid-column-gap: 1rem;
+    grid-template-areas: 'IMAGE TITLE';
+  }
+
+  .image {
+    display: block;
+  }
+
+  .cardItem.normal .image {
+    grid-area: IMAGE;
+  }
+
+  .cardItem.normal .titleWrap {
+    grid-area: TITLE;
+    align-self: center;
+  }
+
+  .cardItem.normal .title {
+    font-size: 1.2rem;
```

Comment:
@rpeterman-gp After reviewing this on my screen, this still feels quite big, especially with the image next to it on mobile. I think `1.1rem` would do the trick.